docs/system-admin-guide/integrations/nextcloud/README.md

```diff
@@ -1,79 +1,96 @@
 ---
 sidebar_navigation:
-  title: Nextcloud integration
+  title: Nextcloud integration setup
   priority: 600
-description: Nextcloud and OpenProject integration
+description: Nextcloud integration setup
 keywords: integrations, apps, Nextcloud
 ---
 
-# OpenProject and Nextcloud integration 
+# OpenProject and Nextcloud integration setup 
 
-The OpenProject and Nextcloud integration will improve the productivity of their enterprise users. It combines the strengths of the market leading content collaboration platform Nextcloud and the leading open source project management software OpenProject.
+The integration between OpenProject and Nextcloud has the potential to improve the productivity of all users. It combines the strengths of Nextcloud, the market-leading collaboration platform, and OpenProject, the leading open source project management software.
 
-The integration is available starting with Nextcloud 20. It enables users to keep an eye on ongoing project activities directly in their Nextcloud instance.
+The integration enables users to keep an eye on ongoing project activities directly in their Nextcloud instance and link their Nextcloud files to OpenProject work packages. To learn more about how to use the integration, please refer to the [Nextcloud user guide](../../../user-guide/nextcloud-integration/)
 
-## Benefits of the integration
+## Step-by-step setup instructions
 
-The first step of the combined effort is the integration of OpenProject in the Nextcloud dashboard. Users can add an OpenProject widget to display latest changes to project's work packages. With this, it offers users a view of ongoing projects and activities.
+The integration is available starting with Nextcloud 20 and OpenProject 12.2. To be able to configure this integration, you need to be an administrator in both Nextcloud and OpenProject instances.
 
-## Step by step instructions
+**1. Add OpenProject integration app and connect the instance** - in Nextcloud
 
-The integration is available starting with Nextcloud 20. It enables users to keep an eye on ongoing project activities directly out of their Nextcloud instance.
+To activate your integration to OpenProject in Nextcloud, navigate to the built-in app store under your user name in _Your apps_. You can use the search field in the top right corner to look for OpenProject integration app. Click the button **Download and enable**.
```

Comment:
What do you mean with "under your user name"?

Reply:
Adding the word "menu" to make avoid potential confusion.